packages/datagateway-common/src/app.types.tsx

```diff
@@ -229,12 +229,3 @@ export interface DownloadCartItem {
   name: string;
   parentEntities: DownloadCartItem[];
 }
```

Comment:
I realise that this is more my mistake than yours, but since you're removing the duplicated `DownloadCart` type here can you also delete the duplicate `DownloadCartItem`? This came about when I was resolving merge conflicts...